scripts/init-database.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import yaml
+import sqlalchemy
+
+values = list()
+
+
+def retreiveValues():
+    stream = open("../.config/bassa/config.yml", "r")
+    docs = yaml.load_all(stream)
+    global values
+    for doc in docs:
+        for k, v in doc.items():
+            for m, n in v.items():
+                values.append(n)
+
+
+def createDatabase():
```

Comment:
@malithsen @padamchopra  We are using camel cases or snake cases for method names?

Reply:
snake case